docker-scripts/env-command.sh

```diff
@@ -8,6 +8,35 @@ echo "--------------------------------------------------------------------------
 
 export PS1='\[\e]0;\w\a\]\n[\#] \[\e[32m\u@Cloud Pak Deployer:\[\e[33m\]\w \e[m\$ ';
 
+if [ -z "$cloud_platform" ]; then
+  echo "cloud_platform is not defined."
```

Comment:
Please use `cp-deploy.sh vault list` to obtain the list of secrets and check if the secrets exist. If they do, extract them using `cp-deploy.sh vault get -vs <secret> -vsf <file>`